core/src/main/scala/org/apache/spark/scheduler/SparkListenerBus.scala

```diff
@@ -71,7 +71,6 @@ private[spark] trait SparkListenerBus
         listener.onNodeUnblacklisted(nodeUnblacklisted)
       case blockUpdated: SparkListenerBlockUpdated =>
         listener.onBlockUpdated(blockUpdated)
-      case logStart: SparkListenerLogStart => // ignore event log metadata
```

Comment:
I think that if this is gonna be posted on the bus it might be better to make it public and `@DeveloperApi` like other events.

Adding new events is not an issue, but posting an event that people can't handle is a little odd.